### ISSUE TITLE
Unable to publish Knowledgebase

### ISSUE THREAD

**Issue by himsoni85** [OfficeDev/microsoft-teams-faqplusplus-app#79]
I was able to deploy the Azure resources using the ARM template and the deployment succeeded without any errors. All resources show up and running as well. When I try to create a new Knowledgebase in the AIMaker portal, I get the error: "No Endpoint keys found".

Error attached.
![error](https://user-images.githubusercontent.com/56127827/66144374-59212180-e626-11e9-9b78-aa79592c7904.jpg)

**Comment by himsoni85** [comment]
I found the fix for the issue (or at least a workaround). If your Resource Group (RG) holding all resources from the template, is in a different location than the one hosting your QnA Cognitive service, then you will face this issue with Keys not being discoverable by the QnA Maker service. 
For example, in my case I had chosen East US as the location for hosting a dedicated RG for all these resources, but the QnA service endpoint was provisioned in West US. I am not sure why that was the case since all three required services: Application Insights, QnA Maker and Azure Search are available in East US as well. However, as soon as I deleted my RG and created it again in West US region, I was able to create the Knowledgebase without any errors.
